packages/pyright-internal/src/common/uri/uriUtils.ts

```diff
@@ -404,3 +404,17 @@ export function getRootUri(isCaseSensitive: boolean): Uri | undefined {
     }
     return undefined;
 }
+
+export function encodeUri(fs: ReadOnlyFileSystem, uri: Uri): string {
+    // Convert to a URI string that the LSP client understands (mapped files are only local to the server).
+    return fs.getOriginalUri(uri).toString();
+}
+
+export function isUri(path: string): boolean {
```

Comment:
Where does this get used? It reminds me of our old problematic logic to try to differentiate between URIs and paths.

Reply:
Nowhere yet (it's in the pylance changes). It will be used in the config options when trying to determine if the execution root on the command line is a URI or not.

Reply:
Specifically, it was necessary when opening a virtual workspace.

Reply:
Maybe it shouldn't be added in this PR then? It would be easier to review if the usage scenarios were more clear.

Reply:
I'll remove it.